### PR TITLE
Us 5

### PR DESCRIPTION
**Pull request by jraniszewski**
Add job profile filter
Add tests to job profile filter
Add directive for job profile filter

### CODE REVIEW

File: src/main/resources/assets/js/application/recruiter/directives/tdprJobProfilesDirecrive.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/recruiter/filters/tdprRecruiterJobProfileFilter'], function (angular, tdprRecruiterModule) {
+    tdprRecruiterModule.directive("jobProfile", function () {
+        return{
+            templateUrl : 'js/application/recruiter/views/tdpr-directive-job-profile.html',
+            link : function($scope, element, attrs){
+                $scope.changeFilter = function(jobProfile){
+                    $scope.jobProfile = jobProfile;
+                }
+            }
```
Comment (msolecka-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprJobProfilesDirecrive.js:9]: We could also set default filter value.

Comment (jraniszewski) [inline src/main/resources/assets/js/application/recruiter/directives/tdprJobProfilesDirecrive.js:9]: Correction done:
IsDev radio default checked;
When job profile variable undefined return all array - checkout filter file.

Comment (msolecka-kainos) [inline src/main/resources/assets/js/application/recruiter/directives/tdprJobProfilesDirecrive.js:9]: Now developer is set in radio box and list without filter is shown, isn't it?

File: src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js [hunk 2]
```diff
@@ -12,6 +14,7 @@ define(['angular', 'application/recruiter/tdprRecruiterModule'
 
         $scope.staticData = [];
         $scope.timeData = {};
+        $scope.jobProfile = "isDev";
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js]: should this not be assigned here from en enum?

File: src/main/resources/assets/js/application/recruiter/filters/tdprRecruiterJobProfileFilter.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+define(['application/recruiter/tdprRecruiterModule'], function (tdprRecruiterModule) {
+    tdprRecruiterModule.filter('jobProfileFilter', function(){
+        return function(persons, jobProfile){
+             if(jobProfile == undefined || jobProfile == ""){
+                return persons;
+             }
+
+            return persons.filter(function(person){
+                return person[jobProfile];
+            })
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/filters/tdprRecruiterJobProfileFilter.js]: missing semicolon

File: src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
     , 'application/recruiter/services/tdprRecruiterGetSlotsTimesService'
     , 'application/recruiter/services/tdprPersonsService'], function (angular, tdprRecruiterModule) {
-    tdprRecruiterModule.controller("tdprWeekTableController", function ($scope, tdprRecruiterGetSlotsTimesService, tdprPersonsService) {
+    tdprRecruiterModule.controller("tdprWeekTableController", function ($scope, tdprRecruiterGetSlotsTimesService, tdprPersonsService, JobProfileEnum) {
         var slotsData = tdprRecruiterGetSlotsTimesService.getSlotsTimes();
         var personsData = tdprPersonsService.fetchPersons();
 
         $scope.staticData = [];
         $scope.timeData = {};
+        $scope.jobProfile = JobProfileEnum["dev"];
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js]: why not use JobProfileEnum.dev?

**Pull request merged**